recipes/cuda-cudart/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "cuda-cudart" %}
+{% set version = "12.0.107" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_cudart/{{ platform }}/cuda_cudart-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: c0e16e9593a69afe4997ae96a8b024e751ebce0254523fd969b08bb028582d23  # [linux64]
+  sha256: d785fd613e35e3761fdb0016a31a56417f570605cbfb9a93ca9dc68be6789188  # [ppc64le]
+  sha256: 5de248c6c203f6c91ce34256bbfc028bc87c1b0c552f9f51edb65e60ad665f63  # [aarch64]
+  sha256: 1cd8516feee068c6d718453e76b0dfcbee27844a35d4f3aa608a3d316190309c  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+  run_exports:                # [linux]
+    strong:                   # [linux]
+      - __glibc >=2.17        # [linux]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
+    - sysroot_{{ target_platform }} 2.17   # [linux]
+    - __glibc >=2.17                       # [linux]
+  run:
+    - sysroot_{{ target_platform }} 2.17   # [linux]
+    - __glibc >=2.17                       # [linux]
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libcudart.so.{{ version }}  # [linux]
+
+outputs:
+  - name: cuda-cudart
+    files:
+      - lib/*.so.*            # [linux]
+      - targets/*/lib/*.so.*  # [linux]
+      - lib/x64/cuda.lib      # [win]
+      - lib/x64/cudart.lib    # [win]
+
+  - name: cuda-cudart-dev
+    requirements:
+      run:
+        - cuda-cudart >={{ version }}
+    files:
+      - lib/*.so               # [linux]
+      - targets/*/lib/*.so     # [linux]
+      - include
+      - lib/x64/cudadevrt.lib  # [win]
+      - bin                    # [win]
+
+  - name: cuda-cudart-static
+    requirements:
+      run:
+        - cuda-cudart-dev >={{ version }}
+    files:
+      - lib/*.a                    # [linux]
+      - targets/*/lib/*.a          # [linux]
+      - lib/x64/cudart_static.lib  # [win]
+
+  - name: cuda-driver-dev
```

Comment:
Can this be a separate recipe instead of being a separate output in the `cuda-cudart` recipe?

Reply:
I'd like to better understand the benefit of separate recipes when the files are from the same source build. We would be duplicating most of the recipe for 1 file

Reply:
Mostly that it would allow rebuilding and publishing new releases of `cuda-driver-dev` separate from `cuda-cudart`, but it's not really a big deal. If they're in the same source build then we should just leave it as is.